spec/controllers/api/posts_controller_spec.rb

```diff
@@ -48,13 +54,13 @@
     let(:mock_post) { double('mock_post') }
 
     before do
-      allow(Post).to receive(:find).and_return(mock_post)
+      allow(Post).to receive(:find_by).and_return(mock_post)
       allow(mock_post).to receive(:attributes=)
       allow(mock_post).to receive(:save!)
     end
 
-    it 'should call Post.find with args' do
-      expect(Post).to receive(:find).with('1')
+    it 'should call Post.find_by with args' do
+      expect(Post).to receive(:find_by).with({ :skater_id=>"1" })
```

Comment:
The hash passed to with is using the old hash syntax, while the other hashes are using the newer one.